draft-ietf-quic-transport.md

```diff
@@ -550,37 +550,45 @@ of protection against off-path attackers.
 ### Initial Packet {#packet-initial}
 
 The Initial packet uses long headers with a type value of 0x7F.  It carries the
-first cryptographic handshake message sent by the client as well as the
-cryptographic messages sent by the server to perform key exchange. The Initial
-packet is protected by Initial keys as described in {{QUIC-TLS}}.
-
-When an an Initial packet is sent by a client which has not previously received
-a Retry packet from the server, it populates the Destination Connection ID field
-with a randomly selected value.  This MUST be at least 8 octets in length. Until
-a packet is received from the server, the client MUST use the same random value
-unless it also changes the Source Connection ID (which effectively starts a new
-connection attempt).  The randomized Destination Connection ID is used to
-determine packet protection keys.
-
-A server sends its first Initial packet in response to a Client Initial.  A
```

Comment:
this is missing?

Reply:
No, I don't think so. They just moved to after the diagram. But maybe I'm confused about which grafs?

Reply:
A server sends its first Initial packet in response to a Client Initial.  A
-server may send multiple Initial packets.  The Cryptographic key exchange could
-require multiple round trips or we might perform retransmissions of this data

On receiving the first Server Initial packet the client uses the Source
-Connection ID supplied by the server as the Destination Connection ID for
-subsequent packets.

Reply:
re-added, but this whole section is kind of clumsy so may try to rewrite.